src/Provider/Instagram.php

```diff
@@ -45,43 +45,88 @@ class Instagram extends OAuth2
     /**
      * {@inheritdoc}
      */
-    protected function initialize()
+    public function getUserProfile()
     {
-        parent::initialize();
+        $parameters = [
+            'fields' => 'id,username,account_type,media_count',
+            $this->accessTokenName => $this->getStoredData($this->accessTokenName),
```

Comment:
since `access_token` is required for any API calls, let's override here a method `initialize` to have something like:

```
$this->apiRequestParameters = [
    $this->accessTokenName => $this->getStoredData($this->accessTokenName),
];

Reply:
Fixed